UI/payments/use_overpayment2.html

```diff
@@ -83,7 +83,7 @@
     <table id="curr_exrate_date_table">
      <tr id="date_row">[% # here goes an input where the date can be written, we can also use a java calendar :). We can use an ajax script to call the Exchange rate of the input date which can be called with the onChange Method          %]
       <th align="right" nowrap id="date_label_column">[% text('Date')%]:</th>
-      <td colspan="2" id="date_column"> [% INCLUDE input element_data=datepaid %] </td>
+      <td colspan="2" id="date_column"> [% datepaid.class = 'date'; INCLUDE input element_data=datepaid %] </td>
```

Comment:
This breaks since #8849 .

```suggestion
      <td colspan="2" id="date_column"> [% datepaid.type = 'date'; INCLUDE input element_data=datepaid %] </td>
```

Reply:
If you want me to wait to merge this until overpayments is working I am fine with that.